mesheryctl/pkg/utils/helpers.go

```diff
@@ -343,3 +343,48 @@ func IsValidSubcommand(available []*cobra.Command, sub string) bool {
 	}
 	return false
 }
+
```

Comment:
@kanishkarj this function is already added, I don't know why did the mesheryctl build passed on this. Can you fix this ?

Reply:
So do I have to remove it?

Reply:
This function is still in use right? I saw in RunE of system subcommand.

Reply:
yes